Dockerfile

```diff
@@ -0,0 +1,9 @@
+# https://docs.docker.com/compose/rails/
+FROM ruby:2.2.0
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev nodejs
```

Comment:
I didn't notice until now that you're using nodejs, if you want latest node you should also add this repo:

```
cat <<EOF > /etc/apt/sources.list.d/nodesource.list
deb https://deb.nodesource.com/node_6.x trusty main
deb-src https://deb.nodesource.com/node_6.x trusty main
EOF
```

And then install nodejs
```
apt-get install -y apt-transport-https lsb-release nodejs
```